src/main/java/appeng/crafting/CraftingJob.java

```diff
@@ -363,9 +364,38 @@ private void setTree( final CraftingTreeNode tree )
 		this.tree = tree;
 	}
 
-	private static class TwoIntegers
+	private void logCraftingJob( String type, Stopwatch timer )
 	{
+		if( AELog.isCraftingLogEnabled() )
+		{
+			final String output = this.output.toString();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename "output" which hides the field declared at line 62. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.orod.org/coding_rules#rule_key=squid%3AHiddenFieldCheck)